app_feup/lib/redux/action_creators.dart

```diff
@@ -287,6 +290,25 @@ ThunkAction<AppState> getRestaurantsFromFetcher(Completer<Null> action) {
   };
 }
 
+ThunkAction<AppState> getCalendarFromFetcher(Completer<Null> action) {
```

Comment:
Do not forget to implement an action to fetch from the local storage and call it on _loadLocalUserInfoToState_